src/services/historyService.js

```diff
@@ -0,0 +1,73 @@
+import axios from "axios";
+
+const getHistory = async (dataDispatch) => {
+  const token = localStorage.getItem("token");
+
+  try {
+    const {
+      data: { history },
+    } = await axios.get(`/api/user/history`, {
+      headers: {
+        authorization: token,
+      },
+    });
+    dataDispatch({ type: "SET_HISTORY", payload: history });
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const postHistory = async (dataDispatch, video) => {
+  const token = localStorage.getItem("token");
+
+  try {
+    console.log("post history");

```

Comment:
Console can be removed .. if not required..

Reply:
ok